multiapps-controller-process/src/main/java/org/cloudfoundry/multiapps/controller/process/Messages.java

```diff
@@ -674,8 +674,9 @@ public class Messages {
     public static final String PROCESS_WAS_DELETED_0 = "Process was deleted: {0}";
     public static final String MODULE_0_WAS_NOT_FOUND = "Module \"{0}\" was not found";
     public static final String DETECTING_LIVE_APPLICATION_ENV = "Detecting live application env...";
-    public static final String USED_EXTENSION_DESCRIPTORS_IDS = "Used extension descriptors with IDs: [{0}]";
-    public static final String NO_EXTENSION_DESCRIPTORS_PROVIDED = "No extension descriptors provided";
+    public static final String EXTENSION_DESCRIPTORS_IN_USE = "Extension descriptors with IDs in use: [{0}]";
```

Comment:
Maybe:
Used extension descriptors for this deployment: ...
and:
Unused extension descriptors for this deployment: ...